unfoldingWord/translationCore
# Auto Assign CC-BY-SA License to door 43 Projects

#### This pull request addresses:
This PR addresses a feature that got removed during the refactor


#### How to test this pull request:
Open any project from door43 online and ensure that after importing the manifest has the field ```"license": "CC BY-SA 4.0"``` and that the license portion of the import stepper did not show.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3423)
